docs/v3.x/concepts/middlewares.md

```diff
@@ -189,6 +189,10 @@ The session doesn't work with `mongo` as a client. The package that we should us
   - `enabled` (boolean): Enable or not `X-Powered-By` header to response. Default value: `true`.
   - `value` (string): The value of the header. Default value: `Strapi <strapi.io>`
 
+::: tip
+`gzip` compression via `koa-compress` uses [Brotli](https://en.wikipedia.org/wiki/Brotli) by default, but is not configured with sensible defaults for most cases. If you experience slow response times with `gzip` enabled, consider disabling Brotli by passing `{br: false}` as an option. You may also pass more sensible params with `{br: { params: { // YOUR PARAMS HERE } }}`
```

Comment:
Can we link to an external resource to provide these params?

Reply:
@derrickmehaffy Wasn't sure where to hunt them down, but it looks like they're part of [Zlib](https://nodejs.org/api/zlib.html#zlib_zlib)? Preference on linking to the [options](https://nodejs.org/api/zlib.html#zlib_class_brotlioptions) doc with the example or the actual [constants](https://nodejs.org/api/zlib.html#zlib_brotli_constants) that are used as params?

Reply:
I see what you mean, it looks like it's part of node itself. For now we can ignore it then.